infra/core/env/postgre-sql/flexible-server.bicep

```diff
@@ -59,17 +74,41 @@ module flexibleServerDeployment 'br/SharedDefraRegistry:db-for-postgre-sql.flexi
     skuName: server.skuName
     activeDirectoryAuth:'Enabled'
     passwordAuth: 'Disabled'
-    enableDefaultTelemetry:false
-    lock: 'CanNotDelete'
+    lock: {
+      kind: 'CanNotDelete'
+    }
     backupRetentionDays:14
     createMode: 'Default' 
-    diagnosticLogCategoriesToEnable: diagnostics.diagnosticLogCategoriesToEnable
-    diagnosticMetricsToEnable: diagnostics.diagnosticMetricsToEnable
-    diagnosticSettingsName:''
-    administrators: []
+    administrators: [
+      {
+        objectId: aadAdminUserMi.outputs.clientId
+        principalName: aadAdminUserMi.outputs.name
+        principalType: 'ServicePrincipal'
+      }
+    ]
     configurations:[]
     delegatedSubnetResourceId : virtual_network::subnet.id
     privateDnsZoneArmResourceId: private_dns_zone.id
-    diagnosticWorkspaceId: ''
   }
 }
+
+var roleDefinitionId = 'acdd72a7-3385-48ef-bd42-f606fba81ae7'
```

Comment:
can you comment what this role name is in code?

Reply:
@DaRoszko done